src/System.Private.ServiceModel/src/System/ServiceModel/SpnEndpointIdentity.cs

```diff
@@ -1,37 +1,31 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.IdentityModel.Claims;
+using System.Runtime;
+using System.Security.Principal; 
+
 namespace System.ServiceModel
 {
     public class SpnEndpointIdentity : EndpointIdentity
-    {
-        private static TimeSpan s_spnLookupTime = TimeSpan.FromMinutes(1);
```

Comment:
Yep. In Desktop, we try to map SPNs with Windows Security Identifiers (SIDs) if possible. 
We don't do this in CoreFx, hence dead code here. Thought it'd look nicer :)